wiki/Beatmap/Pattern/Mania/Jacks/en.md

```diff
@@ -0,0 +1,31 @@
+---
+stub: true
+tags:
+  - jacks
+---
+
+# Jacks
+
+Within specific pattern context, **jacks** are typically used to refer to 3 or more consecutive notes within the same column. They are referred to as **jacks** since the motion required to play them mimics the motion of a jackhammer.
+
+![](img/jack.png "This image displays a jack within it's traditional usage of the word.")
+
+Commonly, the term **jack** can also refer to the general idea of having consecutive notes in the same column. Any of the following specifications can be referred to as just **jacks**.
+
+## Minijack
+
+A **minijack** is a type of **jack** which only features two notes and is the least demanding version of a **jack**.
+
+![](img/minijack.png)
+
+## Longjack
+
+A **longjack** is a more strenuous type of **jack** which typically uses 4 or more consecutive notes. This term may also be used when the same consecutive jumps or hands are used.
+
+![](img/longjack.png)
+
+## Chordjack
+
+**Chordjacks** features a mix of different types of jacks along with chords. Commonly, this term is only used for patterns denser than [quadstreams](mania/streams/quadstreams) and as such a more frequent usage of chords is required.
```

Comment:
Personally I think the definition of a chordjack can be changed a bit. I feel that chordjacks are a completely different mapping style in comparison to quadstreams. You can have a chordjack map that necessarily doesn't use quads for example. You could just add/change:
```suggestion
**Chordjacks** features a mix of different types of jacks along with chords. Commonly, this term is regularly used for patterns denser than [quadstreams](mania/streams/quadstreams) and as such a more frequent usage of chords is required.
```

Reply:
In that case I'd suggest _not_ adding "regularly". You're already using "commonly" at the start of the sentence. Repeating an adverb that's going to mean the same is probably unnecessary.

Reply:
to me both sound a bit off so i'd rather word it like this

```suggestion
**Chordjacks** features a mix of different types of jacks along with chords. This term is typically only used for patterns denser than [quadstreams](mania/streams/quadstreams) and as such a more frequent usage of chords is required.
```